set-web/src/main/java/eu/europeana/set/web/model/vocabulary/Operations.java

```diff
@@ -1,27 +1,24 @@
-package eu.europeana.set.web.model.vocabulary;
-
-public interface Operations {
-
-	//search is also retrieve
-	//crud operations
-	public static final String RETRIEVE = "retrieve";
-	public static final String CREATE = "create";
-	public static final String UPDATE = "update";
-	public static final String DELETE = "delete";
-	
-	//feedback
-	public static final String REPORT = "report";
-	
-	//admin
-	public static final String ADMIN_ALL = "admin_all";
-	public static final String ADMIN_UNLOCK = "admin_unlock";
-	public static final String ADMIN_REINDEX = "admin_reindex"; 
-
-	//user
-	public static final String USER_CREATE = "user_create";
-	public static final String USER_DELETE = "user_delete";
-	
-	//moderation
-	public static final String MODERATION_ALL = "moderation_all";
-	
-}
+package eu.europeana.set.web.model.vocabulary;
+
+public interface Operations {
+
+	//search is also retrieve
+	//crud operations
+	public static final String RETRIEVE = "retrieve";
+	public static final String CREATE = "create";
+	public static final String UPDATE = "update";
+	public static final String DELETE = "delete";

```

Comment:
must extend the Commons-API class and remove dupplicaed constants: https://github.com/europeana/api-commons/blob/ff79e7c6f0a0c71b406f184da1784dd391bc4a51/commons-web/src/main/java/eu/europeana/api/commons/web/model/vocabulary/Operations.java

Reply:
Extended with Operations from api-commons and removed duplicates in set-api

Reply:
The same done for entity-api